ibis/pandas/execution.py

```diff
@@ -170,6 +170,30 @@ def execute_series_natural_log(op, data, scope=None):
     return np.log(data)
 
 
+@execute_node.register(
+    ops.Clip, pd.Series,
+    (pd.Series, float, integer_types, type(None)),
+    (pd.Series, float, integer_types, type(None))
+)
+def execute_series_clip(op, data, lower, upper, scope=None):
+    return data.clip(lower=lower, upper=upper)
+
+
+@execute_node.register(
+    ops.Quantile, pd.Series, float,
```

Comment:
You should be able to cover this in one function by dispatching on `six.string_types + (type(None),)` for the `interpolation` parameter.